app/pages/home-for-user/generic-section.jsx

```diff
@@ -21,7 +22,7 @@ const HomePageSection = React.createClass({
             }}
           ></i>
           <span className="home-page-section__header-label">{this.props.title}</span>
-          <a href="#" className="secret-button" title="Close this section" aria-label="Close this section">×</a>
+          <Link to="#" className="secret-button" title="Close this section" aria-label="Close this section">x</Link>
```

Comment:
I don't think making this a react-router link is necessary since functionally all this is doing is taking you back to the top of the page.